public/styles.css

```diff
@@ -383,51 +384,96 @@ section#pageThree {
 /* #chartBoxInside {
     height: 1000px;
 } */
-#toolBar {
+.toolBar {
     position: absolute;
     display: flex;
     flex-direction: column;
-    height: 100%;
+    height: 754px;
     width: 400px;
     top: 0px;
-    z-index: 10;
-    background-color: #f5f5f5;
-    padding-top: 75px;
+    left: 0;
+    color: #324da6;
+    background-color: #fff;
+    margin-top: 64px;
+    border-bottom: 1px solid #324da6;
+    transition: all 0.2s ease-out;
```

Comment:
Just a quick suggestion, feeling like we can slow it down a little for a smoothier movement; Whether slow it down or not, it's already perfect...

Reply:
Ok :)